apiserver/debuglog_db.go

```diff
@@ -18,7 +18,7 @@ func newDebugLogDBHandler(ctxt httpContext) http.Handler {
 
 func handleDebugLogDBRequest(
 	st state.LogTailerState,
-	reqParams *debugLogParams,
+	reqParams debugLogParams,
```

Comment:
I assume the same thing here, just changing from *struct to struct, along with the tests.

If otherwise, please ask for a deeper review.